_studio/shared/src/mfx_vpp_vaapi.cpp

```diff
@@ -423,14 +424,24 @@ mfxStatus VAAPIVideoProcessing::QueryCapabilities(mfxVppCaps& caps)
     }
 #endif
 
-    /* NB! The code below should to be replaced with querying caps from driver*/
-#if defined(LINUX_TARGET_PLATFORM_BXTMIN) || defined(LINUX_TARGET_PLATFORM_BXT)
-    caps.uMaxWidth  = 8192;
-    caps.uMaxHeight = 8192;
-#else
-    caps.uMaxWidth  = 4096;
-    caps.uMaxHeight = 4096;
-#endif
+    if (m_pipelineCaps.max_output_width && m_pipelineCaps.max_output_height)
+    {
+        caps.uMaxWidth = m_pipelineCaps.max_output_width;
+        caps.uMaxHeight = m_pipelineCaps.max_output_height;
+    }
+    else {
+        /* NB! The code below should to be replaced with querying caps from driver*/
+        VAAPIVideoCORE *hwCore = dynamic_cast<VAAPIVideoCORE *>(m_core);
```

Comment:
See comment from the driver team: https://github.com/intel/media-driver/commit/1479294aa643af67693bad0e5eb6d8eea9ea0b25#commitcomment-29930527. They actually say that Gen8+ supports 16K-32 (16352) and Gen9+ supports 16K (16384). 

I suggest to be as simple as possible here and rely on driver caps. Our open source mediasdk supports Gen8+ platforms. Thus, let's check platforms as >=BDW and hardcode 16K-32 limit here without differentiation between 16K-32.

@onabiull, @dmitryermilov : how do you think?

Reply:
I agree. Let's make it simple and use 16K-32 for >=BDW . 
@IlyaSergeyev , @alariono , since we change behavior here, can you please manually check workability on  16K-32 resolution on BDW ?